test-cloud/pom.xml

```diff
@@ -57,14 +57,9 @@
     <maven.repo.username/>
     <maven.repo.password/>
 
-    <org.kie.server.user>yoda</org.kie.server.user>
-    <org.kie.server.pwd>usetheforce123@</org.kie.server.pwd>
-    <org.kie.workbench.user>adminUser</org.kie.workbench.user>
-    <org.kie.workbench.pwd>adminUser1!</org.kie.workbench.pwd>
-    <org.kie.server.controller.user>controllerUser</org.kie.server.controller.user>
-    <org.kie.server.controller.pwd>controllerUser1!</org.kie.server.controller.pwd>
-    <org.kie.workbench.maven.user>mavenUser</org.kie.workbench.maven.user>
-    <org.kie.workbench.maven.pwd>mavenUser1!</org.kie.workbench.maven.pwd>
+    <kie.app.user>adminUser</kie.app.user>
+    <kie.app.password>adminUser1!</kie.app.password>
+    <kie.app.credentials-secret-name>rhpam-credentials</kie.app.credentials-secret-name>
```

Comment:
Into master branch was merged (see #576) default user yoda, we should use same app user in all branches. users in PR to master branch https://github.com/kiegroup/kie-cloud-tests/pull/576/files#diff-a7e5abb4837467c7dc23f800332d0a34R60-R62

Reply:
well spotted @jakubschwan ! I missed one commit, can you check again?